### PR TITLE
Docs for v2.7.0

### PR DESCRIPTION
**Pull request by francislavoie**
See https://github.com/caddyserver/caddy/compare/v2.6.4...v2.7.0-beta.1

### CODE REVIEW

**Comment by mholt** [comment]
WOW! You've been busy. I'll look at this tomorrow :)

**Review by mholt — APPROVED** [review]
Very nice overall. I have no strong suggestions to make so let's get it in :) Thanks!

**Comment by francislavoie** [comment]
@mholt I'll merge but definitely take a look at the recent commits just so you're caught up 😅

File: src/docs/markdown/command-line.md [hunk 1]
```diff
@@ -495,6 +503,51 @@ The address of this request can be customized using the `--address` flag, or fro
 If you want to stop the current configuration but do not want to exit the process, use [`caddy reload`](#caddy-reload) with a blank config, or the [`DELETE /config/`](/docs/api#delete-configpath) endpoint.
 
 
+### `caddy storage`
+
+Allows export and import of the contents of Caddy's configured data storage.
+
+This is useful when needing to transition from one [storage module](/docs/json/storage/) to another, by exporting from your old one, updating your config, then importing into the new one.
+
+The following command can be used to do this all in one shot, using and old and new config with different configured storage modules, piping the export output into the
```
Comment (mholt) [inline src/docs/markdown/command-line.md:512]: Into the  ...

File: src/docs/markdown/command-line.md [hunk 2]
```diff
@@ -495,6 +503,51 @@ The address of this request can be customized using the `--address` flag, or fro
 If you want to stop the current configuration but do not want to exit the process, use [`caddy reload`](#caddy-reload) with a blank config, or the [`DELETE /config/`](/docs/api#delete-configpath) endpoint.
 
 
+### `caddy storage`
+
+Allows export and import of the contents of Caddy's configured data storage.
```
Comment (mholt) [inline src/docs/markdown/command-line.md:508]: Should we say this is still experimental (liberty to change it)?

**Review by mholt — COMMENTED** [review]
Got it, thanks! Here's a quick thing I noticed

File: src/docs/markdown/command-line.md (hunk 1 above)
Comment (francislavoie) [reply src/docs/markdown/command-line.md:512]: Crap :man_facepalming: I'm too scatterbrained sometimes and half finish sentences. I'll fix it.

File: src/docs/markdown/command-line.md (hunk 2 above)
Comment (francislavoie) [reply src/docs/markdown/command-line.md:508]: Oh right, sure.

**Comment by francislavoie** [comment]
Fixed review comments in ee39c96c788b10579cfd4cbe09143bcfde36b981

**Comment by mholt** [comment]
Thanks!!

**Pull request merged**